pkg/collector/autodiscovery/autoconfig.go

```diff
@@ -226,6 +235,16 @@ func (ac *AutoConfig) getAllChecks() []check.Check {
 // resolveAndRun loads and resolves a given config and schedules the
 // corresponding Check instances.
 func (ac *AutoConfig) resolveAndRun(config check.Config) error {
+	// add default metrics to collect to JMX checks
+	if config.IsJMX() {
+		metrics, ok := ac.name2jmxmetrics[config.Name]
+		if !ok {
+			log.Errorf("JMX check %s doesn't have a metric configuration file", config.Name)
```

Comment:
Is specifying everything in the template still supported?
Looks like if I do (migrating my conf from agent5, or just finding it better), I'll get an error log. Should that be an info?

Reply:
It is still supported, the `AddMetrics` file preserves any metrics currently in `init_config` and only adds new metrics found. The error log might be a bit misleading... I was trying to express that there isn't an _additional_ metrics file present. I'll try to make that more clear 👍

Reply:
Error and Warning should only be printed when user action is necessary, to avoid people ignoring them. Other messages should be info/debug.

Reply:
Noted :)